hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/io/FlinkMergeHandle.java

```diff
@@ -94,6 +94,9 @@ private void deleteInvalidDataFile(long lastAttemptId) {
     final String lastDataFileName = FSUtils.makeDataFileName(instantTime,
         lastWriteToken, this.fileId, hoodieTable.getBaseFileExtension());
     final Path path = makeNewFilePath(partitionPath, lastDataFileName);
+    if (path.equals(oldFilePath)) {
```

Comment:
what happened here? Can we add a comment here explain why?

Reply:
Sure, let me update for it.